Resources/Prototypes/_Mono/companies.yml

```
@@ -110,6 +111,14 @@
   color: "#FC7303"
   image: /Textures/_Mono/Companies/HME.png
 
+  - type: company
```

Should not have a space

```suggestion
- type: company
```